### ISSUE TITLE
Классификация на кастомном датасете

### ISSUE THREAD

**Issue by DarkFlink** [OSLL/sirius-cv#40]
В задаче https://github.com/OSLL/sirius-cv/issues/38 вы создали модель для решения задачи классификации с использованием стандартного датасета. Теперь надо научиться работать с кастомными датасетами:

1. Найти в интернете датасет для решения задачи классификации. Примеры, которые можно взять (можете взять свой):
https://www.kaggle.com/vad13irt/cars-dataset
https://www.kaggle.com/sanikamal/horses-or-humans-dataset
https://www.kaggle.com/sagnikmazumder37/malaria-cell-imagesshuffled-and-split

2. Подгрузить его в ваш блокнот в колабе и убедиться, что изображения считались корректно (например, вывести в консоль одну из картинок).
3. Подстроить модель под данный датасет. В этом этапе есть нюансы, можно почитать как это сделать в примерах например тут:

https://www.reg.ru/blog/mnogoznachnaya-klassifikaciya-s-pomoshchyu-keras/#Dataset
https://blog.keras.io/building-powerful-image-classification-models-using-very-little-data.html

4. Обучить нейронную сеть и построить графики accuracy.

Если застряли на каком-то пункте -- пишите в дискорд.

**Comment by DarkFlink** [comment]
Савинов -- минимум только даталоадер
